File: translations/codenet/uncommented/C++2C/deepseek/Code/s573429823.c
#include <stdio.h>

int main(void)
{
    // Your code here!
    unsigned long n;
    scanf("%lu", &n);
    int flg=0;
    if( n<4 ){
        printf("No\n");
    }else{
        for( int i=0; i<=(n/4); i++)
        {
            for( int j=0; j<=(n/7); j++ )
            {
                if( (i*4+j*7)==n )
                {
                    flg=1;
                    break;
                }
            }
            if(flg) break;
        }
        if(flg)
        {
            printf("Yes\n");
        }else{    
            printf("No\n");
        }
    }
}

File: translations/codenet/uncommented/C++2Go/granite/Code/s573429823.go

package main

import (
    "bufio"
    "fmt"
    "os"
    "strconv"
)

func main() {
    // Your code here!
    var n uint64
    scanner := bufio.NewScanner(os.Stdin)
    if scanner.Scan() {
        var err error
        n, err = strconv.ParseUint(scanner.Text(), 10, 64)
        if err!= nil {
            panic(err)
        }
    }
    flg := false
    if n < 4 {
        fmt.Println("No")
    } else {
        for i := uint64(0); i <= n/4; i++ {
            for j := uint64(0); j <= n/7; j++ {
                if i*4+j*7 == n {
                    flg = true
                    break
                }
            }
            if flg {
                break
            }
        }
        if flg {
            fmt.Println("Yes")
        } else {
            fmt.Println("No")
        }
    }
}
